docs/pages/eas/workflows/syntax.mdx

```diff
@@ -384,6 +384,33 @@ jobs:
       prod: boolean # optional
 ```
 
+#### `fingerprint`
+
+Calculates fingerprint of Android and iOS builds.
+
+```yaml
+jobs:
+  my_job:
+    # @info #
+    type: fingerprint
+    # @end #
+```
+
+> **warning** This job type only supports [CNG (managed)](/workflow/continuous-native-generation/) workflows. If you commit your **android** or **ios** directories, the fingerprint job won't work.
+
+You can then use the results of the fingerprint job when looking for existing build:
+
+```yaml
+jobs:
+  get_ios_build:
+    name: Check for existing ios build
+    needs: [fingerprint]
+    type: get-build
+    params:
+      fingerprint_hash: ${{ needs.fingerprint.outputs.ios_fingerprint_hash }}
+      profile: production
+```
```

Comment:
Instead of showing this, we should show the outputs of the fingerprint job: `android_fingerprint_hash` and `ios_fingerprint_hash`. Should match the outputs section of the `build` job.

Then in the examples guide we can show the `fingerprint` job, and show another `get-build` job using its output.